docs/loading-in-profiles.md

```diff
@@ -0,0 +1,45 @@
+# Loading in profiles from various sources
+
+Profiles can be loaded in perf.html from many different sources.
+
+### Online Storage
+
+> `https://perf-html.io/public/{HASH}`
+
+Profiles can be stored in online data store. The hash is used to retrieve it. This is where profiles go when clicking the "Share..." button. Here is an example bash script to programmatically upload profiles:
+
+```bash
+uploadprofile() {
+  gzip $1
+  curl 'https://profile-store.appspot.com/compressed-store' --compressed --data-binary @$1.gz | awk '{print "Hosted at: https://perf-html.io/public/"$1}'
+}
```

Comment:
I'd use something like this instead:
```bash
uploadprofile() {
  gzip -c "$1" | curl 'https://profile-store.appspot.com/compressed-store' --compressed --data-binary @- | awk '{print "Hosted at: https://perf-html.io/public/"$1}'
}
```
Main reason: this doesn't change the source file.
Also note the quotes to allow weird names (including spaces in name). 

I tried it myself so you can use it directly.